onnxruntime/server/core/metric_registry.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include <prometheus/registry.h>
+
+namespace onnxruntime {
+namespace server {
+struct MetricRegistry {
+  // returns the global metric registry instance
+  // use this handle to get the instance to increment/count requests
+  static MetricRegistry& Get() {
+      static MetricRegistry metricRegistry;
+      return metricRegistry;
+  }
+
+  // Store quantile buckets 
+  static prometheus::Histogram::BucketBoundaries TimeBuckets () {
+    // Anything greater than 10s we should bucket into +inf
+    return {0, 50, 100, 250, 500, 1000, 1000, 2500, 5000, 10000};
+  }
+
+  static prometheus::Histogram::BucketBoundaries ByteBuckets() {
+    // get megabyte boundaries?
+    return {
+      0, 500000, 1000000, 2000000, 3000000, 4000000,
```

Comment:
when we talk about 1K, I would expect 1024 other than 1000. Also we do see some input image could be up to 18MiB.

Reply:
True, I will convert to bytes